docs/blog/posts/2024/srlinux-asymmetric-routing.md

```diff
@@ -0,0 +1,1801 @@
+      exec:
+        - ip link add bond0 type bond mode 802.3ad
+        - ip link set eth1 down 
+        - ip link set eth2 down 
+        - ip link set eth1 master bond0
+        - ip link set eth2 master bond0
+        - ip addr add 172.16.10.2/24 dev bond0
+        - ip link set eth1 up
+        - ip link set eth2 up
+        - ip link set bond0 up
+        - ip route add 172.16.20.0/24 via 172.16.10.254
+    s3:
+      kind: linux
+      image: ghcr.io/srl-labs/network-multitool
+      exec:
+        - ip addr add 172.16.20.3/24 dev eth1
+        - ip route add 172.16.10.0/24 via 172.16.20.254
+  links:
+    - endpoints: ["leaf1:e1-1", "spine1:e1-1"]
+    - endpoints: ["leaf1:e1-2", "spine2:e1-1"]
+    - endpoints: ["leaf2:e1-1", "spine1:e1-2"]
+    - endpoints: ["leaf2:e1-2", "spine2:e1-2"]
+    - endpoints: ["leaf3:e1-1", "spine1:e1-3"]
+    - endpoints: ["leaf3:e1-2", "spine2:e1-3"]
+    - endpoints: ["leaf4:e1-1", "spine1:e1-4"]
+    - endpoints: ["leaf4:e1-2", "spine2:e1-4"]
+    - endpoints: ["leaf1:e1-3", "s1:eth1"]
+    - endpoints: ["leaf2:e1-3", "s2:eth1"]
+    - endpoints: ["leaf3:e1-3", "s2:eth2"]
+    - endpoints: ["leaf4:e1-3", "s3:eth1"]
+```
+
+/// admonition | Credentials
+    type: subtle-note
+As usual, Nokia SR Linux nodes can be accessed with `admin:NokiaSrl1!` credentials and the host nodes use `user:multit00l`.
+///
+
+The end goal of this post is to ensure that server s1 can communicate with both s2 (same subnet) and s3 (different subnet) using an asymmetric routing model. To that end, the following IPv4 addressing is used (with the IRB addressing following a distributed, anycast model):
+
+|      Resource       |    IPv4 scope    |
+| :-----------------: | :--------------: |
+|      Underlay       | 198.51.100.0/24  |
+| `system0` interface |   192.0.2.0/24   |
+|      VNI 10010      |  172.16.10.0/24  |
+|      VNI 10020      |  172.16.20.0/24  |
+|      server s1      |  172.16.10.1/24  |
+|      server s2      |  172.16.10.2/24  |
+|      server s3      |  172.16.20.3/24  |
+| `irb0.10` interface | 172.16.10.254/24 |
+| `irb0.20` interface | 172.16.20.254/24 |
+
+## Reviewing the asymmetric routing model
+
+When routing between VNIs, in a VXLAN fabric, there are two major routing models that can be used - asymmetric and symmetric. Asymmetric routing, which is the focus of this post, uses a `bridge-route-bridge` model, implying that the ingress leaf bridges the packet into the Layer 2 domain, routes it from one VLAN/VNI to another and then bridges the packet across the VXLAN fabric to the destination.
+
+Such a design naturally implies that both the source and the destination IRBs (and the corresponding Layer 2 domains and bridge tables) must exist on all leafs hosting servers that need to communicate with each other. While this increases the operational state on the leafs themselves (ARP state and MAC address state is stored everywhere), it does offer configuration and operational simplicity.
```

Comment:
I admit I did a quick read, but I failed to see the explanation why this mode is simpler config and oper-wise?

Would be good to mention that this mode has scale considerations as you duplicate state of each BD, which, in larger fabrics, may play a critical role.

Reply:
already added how mac and arp state is compounded in such designs.